src/devtools/client/inspector/rules/models/fronts/rule.ts

```diff
@@ -41,12 +44,24 @@ export class RuleFront {
     return this._rule.cssText;
   }
 
-  get selectorText() {
-    return this._rule.selectorText;
+  get cleanedSelectorText() {
```

Comment:
I don't understand why this function is needed, or what a "clean selector text" is.

It looks kind of like this and the getter below only exist to add a space after a comma. Is that...right?

Reply:
It's pretty stupid, really:

- We apparently split the original selector text on commas and trim each entry
- But this code wants a string that's effectively the same as the original except with an extra space after each comma.

Honestly I suspect we could probably just use the original selector text string as-is and it'd be essentially the same result, but I was trying to preserve the existing behavior here.

Reply:
> Honestly I suspect we could probably just use the original selector text string as-is and it'd be essentially the same result, but I was trying to preserve the existing behavior here.

Seems like this would be worth doing.